src/app/operators/components/operator/operator.component.ts

```diff
@@ -2,28 +2,56 @@ import {
   Component,
   Input,
   OnInit,
-  ChangeDetectionStrategy
-} from "@angular/core";
-import { OperatorDoc } from "../../../../operator-docs/operator.model";
+  ChangeDetectionStrategy,
+  Inject,
+  InjectionToken
+} from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+import { SeoService } from '../../../services/seo.service';
+import { OperatorDoc } from '../../../../operator-docs/operator.model';
+import 'rxjs/add/operator/pluck';
```

Comment:
Maybe it'll be better to use only lettable operators when this project is already using RxJS `5.5.2`?

Reply:
hi @martinsik and thanks for popping in to say hi! :) thx for reviewing this PR.